gammapy/makers/utils.py

```diff
@@ -185,11 +217,58 @@ def _map_spectrum_weight(map, spectrum=None):
     return map * weights.reshape(shape.astype(int))
 
 
+def _evaluate_bkg(pointing, ontime, bkg, geom, use_region_center, obstime, d_omega):
+    """
+    Evaluate the background IRF on a given geometry.
+
+    Parameters
+    ----------
+    pointing :  `~gammapy.data.FixedPointingInfo` or `~astropy.coordinates.SkyCoord`
+        Observation pointing.
+    ontime : `~astropy.units.Quantity`
+        Observation ontime. i.e. not corrected for deadtime
```

Comment:
Why the deadtime is not applied? At the end, it should be, is not it?

Reply:
That is how it was previously implemented. 
Now to answer your question, I don't know. It depends on how the background IRF is defined. If it is "the number of background counts that we expect to be seen when observing a certain duration" then the answer is no.
If it is "The number of background events that could trigger the camera if it is available during a certain duration", then the livetime needs to appear at some point.

I would say the first definition is more logical. (Of course, if the ontime and the duration of GTIs are not the same, this would need to be accounted for).